pkgs/data/icons/fluent-icon-theme/default.nix

```
@@ -1,73 +1,60 @@
-{ lib
-, stdenvNoCC
-, fetchFromGitHub
-, gtk3
-, hicolor-icon-theme
-, jdupes
-, roundedIcons ? false
-, blackPanelIcons ? false
-, colorVariants ? []
-,
-}:
-let
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  gtk3,
+  hicolor-icon-theme,
+  jdupes,
+  roundedIcons ? false,
+  blackPanelIcons ? false,
+  colorVariants ? [],
+}: let
```

```suggestion
{ lib
, stdenvNoCC
, fetchFromGitHub
, gtk3
, hicolor-icon-theme
, jdupes
, roundedIcons ? false
, blackPanelIcons ? false
, colorVariants ? []
}:

let
```
nixpkgs is not using alejandra and it is incompatible with nixpkgs-fmt and nix-fmt.

---

Was not aware of this fact. 
Will make sure to remember this when contributing in the future!